src/Channel.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Created by ken-linux2 on 12/07/16.
+ */
+public class Channel {
+
+    private String name;
+    private String channel_campaign_id;
+    private String channel_adgroup_id;
+    private String channel_account_id;
+    private String channel_ad_id;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        if (name == null) {
+            throw new RuntimeException("name cannot be null");
+        }
+        this.name = name;
+    }
+
+    public String getChannel_campaign_id() {
+        return channel_campaign_id;
+    }
+
+    public void setChannel_campaign_id(String channel_campaign_id) {
+        if(channel_campaign_id==null)
```

Comment:
You checkNotNull(T). You'll need to add a new dependency to build.gradle.
Try to do it as an exercise and let me know if you need help.